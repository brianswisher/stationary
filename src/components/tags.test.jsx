import React from "react";
import renderer from "react-test-renderer";

import Tags from "./tags";

const data = ["a", "b"];

describe("Tags", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Tags data={data} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
