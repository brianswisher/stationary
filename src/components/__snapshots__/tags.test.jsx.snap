// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tags renders correctly 1`] = `
<span
  className="tags"
>
  <span
    className="tag"
  >
    a
  </span>
  <span
    className="tag"
  >
    b
  </span>
</span>
`;
